src/main/java/net/mcreator/ui/modgui/ArmorGUI.java

```diff
@@ -482,25 +485,21 @@ public ArmorGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		enableHelmet.addActionListener(event -> {
 			textureHelmet.setEnabled(enableHelmet.isSelected());
 			helmetName.setEnabled(enableHelmet.isSelected());
-			helmetItemRenderType.setEnabled(enableHelmet.isSelected());
```

Comment:
Why should it be kept if the item will not be generated? It might confuse the user.

Reply:
Because all other fields are also kept enabled, except the simple fields of texture and name.

You can make all fields disable on this, but I think this is a bit out of scope this this particular PR